python/plugins/processing/core/GeoAlgorithm.py

```diff
@@ -194,7 +193,7 @@ def processBeforeAddingToModeler(self, alg, model):
 
     # =========================================================
 
-    def execute(self, progress=SilentProgress(), model=None):
+    def execute(self, feedback=QgsProcessingFeedback(), model=None):
```

Comment:
I think it is better to have None as the default value for feedback arg, to avoid surprises: http://effbot.org/zone/default-values.htm